### ISSUE TITLE
Zombi  Pygame

### ISSUE THREAD

**Issue by J1Man** [Syndwar/lua-sdl-engine#1]
Hello Syndwar,

Are you the creator of the Zombi game linked below?  Zombi is an amazing game and my childhood favorite.  I would appreciate if you can update the links for the game and its source code.  All links are dead.  

It is even better if you can create a repository for Zombi on github.  You did an amazing thing by porting it to windows.

http://pygame.org/project-Zombi-2757-4479.html
http://dev.pygame.org/project-Zombi-2757-4479.html

I am sorry because I am contacting you here.  Pygame website does not allow registrations.  That's why I couldn't PM you there.

**Comment by Syndwar** [comment]
Yes, I am. The links are dead because I was forced to remove them in may. It took 5 years for Ubisoft to learn about this project and use their ban hammer on me.
The source code is 100% copyright free - it was written by me just looking at the game, but the graphics and sounds, taken from zx-spectrum emulator are not free to use, I guess)

**Comment by Syndwar** [comment]
"amazing game and my childhood favorite". I liked it so much that I decided it to be my first finished working game. 5 years ago I tried to find someone to redraw the graphics but failed, maybe I will try to do this again: redraw the graphics, review the old code, hide the old title, and resurrect the game in Ubisoft free form)

**Comment by J1Man** [comment]
Oh no.  I am really sorry to hear that you had to take it down per Ubisoft's request.

Is it possible to modify the source code so that the game engine reads and parses the game assets from the ZX spectrum game rom (Zombie.tap) or from dos version of the game?
 
That's how the game engines that are included in the opesource ScummVM poject behave.  They just a put a note saying that you need the original game files to run the game.   Since the engine itself does not include any game assets, companies are not able to take down the ScummVM game engine.

**Comment by Syndwar** [comment]
I'm not sure that using game rom would be possible. But I'll think of something) Working in the game development leaves little free time for other projects but I'll try.

**Comment by J1Man** [comment]
Thanks Snydwar.  I know that it is a hobby project for you, so I am aware that it won't happen right away.
